catalyst_voices/packages/catalyst_voices_services/lib/src/registration/registration_service.dart

```diff
@@ -37,6 +74,31 @@ final class RegistrationService {
     );
   }
 
+  // TODO(damian-molinski): to be implemented
+  Future<CardanoWalletDetails> recoverCardanoWalletDetails(
+    SeedPhrase seedPhrase,
```

Comment:
There is no direct correlation between the wallet (extension) and the seed phrase. Seed phrase isn't the ID of the wallet, it's the "ID" of the user. The user has a wallet registered in a wallet extension. The user might take their wallet to a different extension and keep using it as before. 

We're unable of recovering wallet, we can ask the user to link it again but we don't need the seed phrase for it. I see this method is marked as TODO but it needs to be rethinked

Reply:
Agree, i'll update comment in this TODO and leave a note. UI suggests that we have to display connected wallet and balance. If that something we can not do with only seed phrase then it have to be changed.

Eitherway i want to wait till we're have working endpoint for this lookup to see what data we can get